k2/csrc/fsa_algo_test.cc

```diff
@@ -19,10 +19,13 @@
 namespace k2 {
 
 TEST(FsaAlgo, Connect) {
-  std::vector<Arc> arcs = {
-      {0, 1, 1}, {0, 2, 2}, {1, 3, 3}, {1, 6, 6},
-      {2, 4, 2}, {2, 6, 3}, {5, 0, 1},
-  };
+  std::vector<Arc> arcs = {{0, 1, 1},
+                           {0, 2, 2},
+                           {1, 3, 3},
+                           {1, 6, 6},
+                           {2, 4, 2},
+                           {2, 6, 3},
+                           {5, 0, 1}, };
```

Comment:
Guys.... the `arc_indexes` arg of the constructor of Fsa is not needed because the constructor can work out what value the arc_indexes array should have.   There can be a version of the constructor where arc_indexes is not provided. This will save typing in tests.  I have said this several times before; if you don't understand what I mean, or you disagree, please say so!

Reply:
Will add such constructor with another PR.